### ISSUE TITLE
Invalid order function for sorting bug

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#2173]
```lua
2019-12-12 11:08:57: ********
2019-12-12 11:08:57: 11:08:57 ERROR:      prop: Error while notifying a watcher: ...king/Github/CommandPost/src/extensions/cp/ui/axutils.lua:185: invalid order function for sorting
stack traceback:
	[C]: in function 'table.sort'
	...king/Github/CommandPost/src/extensions/cp/ui/axutils.lua:185: in function 'cp.ui.axutils.children'
	...t/src/extensions/cp/apple/finalcutpro/viewer/InfoBar.lua:34: in field 'matches'
	...st/src/extensions/cp/apple/finalcutpro/viewer/Viewer.lua:122: in local 'verifyFn'
	...king/Github/CommandPost/src/extensions/cp/ui/axutils.lua:682: in upvalue 'isInvalid'
	...king/Github/CommandPost/src/extensions/cp/ui/axutils.lua:708: in function 'cp.ui.axutils.cache'
	(...tail calls...)
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:510: in function <...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:507>
	(...tail calls...)
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:1116: in field '_get'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:510: in method 'get'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:1531: in field '_get'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:510: in function <...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:507>
	(...tail calls...)
	.../CommandPost/src/plugins/finalcutpro/viewer/overlays.lua:651: in field 'update'
	.../CommandPost/src/plugins/finalcutpro/viewer/overlays.lua:1573: in field 'fn'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:1234: in function <...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:1278: in method '_notify'
	...cking/Github/CommandPost/src/extensions/cp/prop/init.lua:881: in method 'update'
	...st/src/extensions/cp/apple/finalcutpro/viewer/Viewer.lua:184: in local 'action'
	...g/Github/CommandPost/src/extensions/cp/deferred/init.lua:51: in function <...g/Github/CommandPost/src/extensions/cp/deferred/init.lua:49>
2019-12-12 11:08:57: ********
```

**Comment by latenitefilms** [comment]
Valerio D'Andrassi reports:

```lua
CommandPost v1.0.5 (Build: 5722)

Build Date: � � � � � � � � � � Jan� 6 2020, 00:14:11
macOS Version:� � � � � � � � � 10.14.6
CommandPost Locale: � � � � � � en
Current Keyboard Layout:� � � � ABC
Final Cut Pro Path: � � � � � � /Applications/Final Cut Pro.app
Final Cut Pro Version:� � � � � 10.4.6
Final Cut Pro Language: � � � � en
Developer Mode: � � � � � � � � false

2020-01-05 16:51:19: 2020-01-05 16:51:19: -- Loading /Applications/CommandPost.app/Contents/Resources/extensions/cp/init.lua
2020-01-05 16:51:19: -- Loading extension: hs.uielement
2020-01-05 16:51:19: 16:51:19 � � � � cp: � � Loading Plugins...
2020-01-05 16:51:25: 16:51:25 ** Warning: � plugins: Slow Plugin: finalcutpro.setup.scan (3.415048)
2020-01-05 16:51:25:� � � � � � � � � cp: � � Plugins Loaded.
2020-01-05 16:51:25:� � � � � � � � � � � � � Startup Time: 4.85645 seconds
2020-01-05 16:51:25: -- Done.
2020-01-05 16:58:50: ********
2020-01-05 16:58:50: 16:58:50 ERROR:� � � prop: Error while notifying a watcher: ...Post.app/Contents/Resources/extensions/cp/ui/axutils.lua:185: invalid order function for sorting
stack traceback:
	[C]: in function 'table.sort'
	...Post.app/Contents/Resources/extensions/cp/ui/axutils.lua:185: in function 'cp.ui.axutils.children'
	...es/extensions/cp/apple/finalcutpro/viewer/ControlBar.lua:42: in field 'matches'
	...ources/extensions/cp/apple/finalcutpro/viewer/Viewer.lua:123: in local 'fn'
	...st.app/Contents/Resources/extensions/hs/fnutils/init.lua:108: in function 'hs.fnutils.ifilter'
	(...tail calls...)
	...ources/extensions/cp/apple/finalcutpro/viewer/Viewer.lua:62: in upvalue 'findViewersUI'
	...ources/extensions/cp/apple/finalcutpro/viewer/Viewer.lua:81: in function <...ources/extensions/cp/apple/finalcutpro/viewer/Viewer.lua:80>
	(...tail calls...)
	...Post.app/Contents/Resources/extensions/cp/ui/axutils.lua:709: in function 'cp.ui.axutils.cache'
	(...tail calls...)
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:510: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:507>
	(...tail calls...)
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1116: in field '_get'
	...
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:881: in method 'update'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:812: in field 'fn'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in function <...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234>
	[C]: in function 'xpcall'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1234: in upvalue '_notifyWatchers'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:1278: in method '_notify'
	...dPost.app/Contents/Resources/extensions/cp/prop/init.lua:881: in method 'update'
	...CommandPost.app/Contents/Resources/extensions/cp/app.lua:1243: in function <...CommandPost.app/Contents/Resources/extensions/cp/app.lua:1242>
2020-01-05 16:58:50: ********
```

**Comment by latenitefilms** [comment]
Fixed in #2263
